CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1 @@
+Fix: avoid duplicated attributes in arrays in POST v2/subscriptions (#2101)
```

Comment:
```suggestion
Fix: avoid duplicated attributes and metadata in arrays in POST /v2/subscriptions (#2101)
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4025/commits/81e67f5fb86e4e61d4879e6ee8924b88fdd5d5cb